### PR TITLE
Website Redesign: New product images

### PR DESCRIPTION
**Pull request by JasonMun7**
### Summary 

This PR adds the new product images for the Product Page, effectively omitting the Floating Images component. (Keeping it for the future if needed!)

- [x] Added new Product Images in a new static folder in the public/images/public directory!
- [x] Removed the Floating images component the use of cn class! 
- [x] Updated products json to simply take a single image and a new image dimension!

### Notion/Figma Link 

[Redo Image Layout](https://www.notion.so/Products-Page-Redo-image-layout-1510ad723ce1804e8caffdca4a915fdd?pvs=4)

### Test Plan 

https://github.com/user-attachments/assets/33609548-1c52-480b-9b91-941814cc50b9

### Notes

- Planning to fix margins and spacing in a future PR using a Wrapper Component!

### CODE REVIEW

**Review by andrew032011 — APPROVED** [review]
Thanks! A few questions: 
1. Any unused images that we can delete now?
2. The images look kind of large (in size). Do you think it's possible to keep them within the 300 KB policy we've been following, without decreasing quality? I noticed they took awhile to load in.

File: new-dti-website/components/products/products.json [hunk 1]
```diff
@@ -18,20 +18,8 @@
...
-          "right": "15%",
-          "width": "60%",
-          "direction": "down"
-        }
-      ]
+      "productImage": "/images/products/static/cureviews.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "courseplan",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:25]: can we make the alt text a bit descriptive?

For example this is what Apple does for similar product mockups:
Screenshot 2024-12-05 at 10 03 58 AM


 I'll provide some suggestions in the review

File: new-dti-website/components/products/products.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
       "link": "https://cureviews.org/",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:7]: alt for CU reviews:

> A laptop next to a phone, displaying the search page and the search results page of the CU Reviews website

File: new-dti-website/components/products/products.json (hunk 1 above)
Comment (Bookie0) [inline new-dti-website/components/products/products.json:25]:
```suggestion
      "alt": "An desktop computer next to a laptop, displaying the planning page and the profile page of the Course Plan website",
```

File: new-dti-website/components/products/products.json [hunk 3]
```diff
@@ -51,29 +39,8 @@
...
-          "width": "30%",
-          "direction": "up",
-          "delay": true
-        }
-      ]
+      "productImage": "/images/products/static/courseplan.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "queue me in",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:46]:
```suggestion
      "alt": "2 phones side by side, displaying the list of available consulting hours screen and the join queue screen of the Queue Me In website",
```

File: new-dti-website/components/products/products.json [hunk 4]
```diff
@@ -93,28 +60,8 @@
...
-          "bottom": "7%",
-          "right": "-11%",
-          "direction": "down"
-        }
-      ]
+      "productImage": "/images/products/static/qmi.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "design at cornell",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:67]:
```suggestion
      "alt": "A browser displaying the hero section of the Design @ Cornell website",
```

File: new-dti-website/components/products/products.json [hunk 5]
```diff
@@ -123,44 +70,15 @@
...
-          "right": "-16%",
-          "width": "43%",
-          "direction": "right"
-        }
-      ]
+      "productImage": "/images/products/static/dcc.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "zing",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:84]:
```suggestion
      "alt": "A desktop computer next to a tablet, displaying the landing page and questionnaire page of the Zing website",
```

**Comment by andrew032011** [comment]
CornellGo hasn't launched yet: I think it's fine to keep it without the button now and then sub it in once they launch? 

either way, filed a ticket to track: https://www.notion.so/Add-CornellGo-App-Store-Google-Play-Store-Link-To-Product-Page-1530ad723ce1805ea331d7a6253421a5?pvs=4

File: new-dti-website/components/products/products.json [hunk 6]
```diff
@@ -180,29 +98,8 @@
...
-          "right": "-19%",
-          "width": "80%",
-          "direction": "up"
-        }
-      ]
+      "productImage": "/images/products/static/zing.png",
+      "productImageDimensions": 1500
     },
     {
       "alt": "cuapts",
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:105]:
```suggestion
      "alt": "A laptop next to a phone, displaying home page and an address page of the CU Apts website",
```

File: new-dti-website/components/products/products.json [hunk 7]
```diff
@@ -221,19 +118,8 @@
...
-          "bottom": "4%",
-          "right": "4%",
-          "direction": "down"
-        }
-      ]
+      "productImage": "/images/products/static/cuapts.png",
+      "productImageDimensions": 1500
     }
   ],
   "upcoming": [
```
Comment (Bookie0) [inline new-dti-website/components/products/products.json:125]: Alt for Carriage:

> A monitor displaying a blurry screen of the Carriage website with the words "Coming Soon" overlaid on top

**Review by Bookie0 — CHANGES_REQUESTED** [review]
nice thank you! let's also try to improve the alt text of these images while we're at it :)

**Comment by JasonMun7** [comment]
> Thanks! A few questions:
> 
> 1. Any unused images that we can delete now?
> 2. The images look kind of large (in size). Do you think it's possible to keep them within the 300 KB policy we've been following, without decreasing quality? I noticed they took awhile to load in.

Yes I can remove the floating images within the public folder! Also, you right! I'll try to see if I can decrease their size! Thanks for that reminder Andrew!

**Review by Bookie0 — APPROVED** [review]
yay thanks!

**Pull request merged**